client/src/Components/Common/Popover/index.js

```diff
@@ -68,8 +143,9 @@ class PopoverComponent extends React.Component {
             color={color}
           />
         ) : (
-          <p>{this.props.children}</p>
+          <>{this.props.children}</>
```

Comment:
Since you've destructured props further up, you could put children as put of this destructuring to keep it all consistent